netebpfext/net_ebpf_ext.c

```diff
@@ -539,6 +581,9 @@ _net_ebpf_ext_register_wfp_callout(_Inout_ net_ebpf_ext_wfp_callout_state_t* cal
     callout_register_state.flags = 0;
 
     status = FwpsCalloutRegister(device_object, &callout_register_state, &callout_state->assigned_callout_id);
+    if (WFP_ERROR(status, ALREADY_EXISTS)) {
```

Comment:
nit: in what case can it already exist?

Reply:
reboot without driver being unloaded - such as kernel crash or power failure.

Reply:
Come to think of it, since `FWPM_*_FLAG_PERSISTENT` is not passed for any objects; none of these objects should remain after a reboot. You were likely hitting this during your intermediate testing. I think this check can be safely removed.